concent_api/core/models.py

```diff
@@ -43,3 +63,13 @@ def __str__(self):
     class Meta:
         verbose_name        = ('ReceiveOutOfBand status')
         verbose_name_plural = ('ReceiveOutOfBand statuses')
+
+
+class MessageAuth(Model):
+    """
+    This class is used to store provider and requestor keys for message exchange related to given initial message.
+    """
+
+    message              = OneToOneField(Message, related_name = 'auth')
+    provider_public_key  = Base64Field()
+    requestor_public_key = Base64Field()
```

Comment:
Please set max length for the text fields. We don't want anyone to feed us gigabytes of data in the key field. We do have validations against that in the views but having database constraints as the last line of defense is always a good idea.

Reply:
:+1: